welcome/oke_about.adoc

```diff
@@ -215,8 +215,8 @@ derived from the istio.io open source project, is not supported in {oke}. Also,
 the Kourier Ingress Controller found in OpenShift Serverless is not supported
 on {oke}.
 
-=== {sandboxed-containers-first}
-{oke} does not include {sandboxed-containers-first}. Use {product-title} for this support.
+=== {osc}
+{oke} does not include {osc}. Use {product-title} for this support.
```

Comment:
For improved clarity, rather than say "this support" I would consider listing the product so there is no ambiguity. 

Maybe something along the lines of: {oke} does not include {osc}. Use {product-title} to support {osc} .